### ISSUE TITLE
Exception on startup

### ISSUE THREAD

**Issue by drmatthews** [flimlib/flimj-ui#23]
I installed the ImageJ plugin into Fiji as in the installation instructions but I get an exception when starting the plugin. I am running Fiji on macOS 11.2.1.

```
(Fiji Is Just) ImageJ 2.1.0/1.53c; Java 1.8.0_202 [64-bit]; Mac OS X 10.16; 88MB of 8446MB (1%)
 
java.lang.NoClassDefFoundError: javafx/embed/swing/JFXPanel
	at java.lang.Class.getDeclaredConstructors0(Native Method)
	at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
	at java.lang.Class.getConstructor0(Class.java:3075)
	at java.lang.Class.newInstance(Class.java:412)
	at org.scijava.plugin.PluginInfo.createInstance(PluginInfo.java:304)
	at org.scijava.command.CommandInfo.createInstance(CommandInfo.java:247)
	at org.scijava.command.CommandModule.instantiateCommand(CommandModule.java:248)
	at org.scijava.command.CommandModule.<init>(CommandModule.java:98)
	at org.scijava.command.CommandInfo.createModule(CommandInfo.java:325)
	at org.scijava.module.DefaultModuleService.createModule(DefaultModuleService.java:167)
	at org.scijava.module.DefaultModuleService.run(DefaultModuleService.java:206)
	at org.scijava.module.DefaultModuleService.run(DefaultModuleService.java:197)
	at org.scijava.module.DefaultModuleService.run(DefaultModuleService.java:182)
	at net.imagej.legacy.LegacyService.runLegacyCompatibleCommand(LegacyService.java:302)
	at net.imagej.legacy.DefaultLegacyHooks.interceptRunPlugIn(DefaultLegacyHooks.java:166)
	at ij.IJ.runPlugIn(IJ.java)
	at ij.Executer.runCommand(Executer.java:150)
	at ij.Executer.run(Executer.java:68)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: javafx.embed.swing.JFXPanel
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```

**Comment by drmatthews** [comment]
This turned out to be related to Fiji bundling `adoptopenjdk-8.jdk` with the mac os version which does not include JavaFX. According to this issue https://github.com/fiji/fiji/issues/240 Fiji should now be bundled with Azul Java 8 JDK+FX 7, however, I have found this not to be the case for the mac os version which still includes adoptopenjdk. I got this working using the no JRE version of Fiji along with a system Java setup which includes JavaFX.
